acb.yaml

```diff
@@ -0,0 +1,2 @@
+steps:
```

Comment:
Would be good to add some comments that explain the file's purpose, and also add a section to the README.md that explains its use.

Also, is "acb.yaml" a standard across several examples? Wondering if maybe it can use a more self-evident filename.

Reply:
@mmacy updated docs slightly, let me know if you have better wording/thoughts. Feel free to push to the PR.

`acb.yaml` is the standard we'd been using for a while (Azure Container Builder yaml). But I think it might end up changing longer term as Steve mentioned before.

Reply:
acb.yaml was the initial format when we were focusing on azure container builder. With tasks being our focus, which can achieve build, push as we well, I'd like to use `acr-task.yaml` as the generic filename. We don't yet have a default value, so you still need to specify `-f acr-task.yaml`, but lets use this file name for the preview release.